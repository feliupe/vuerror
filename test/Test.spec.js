const { mount } = require('@vue/test-utils')
const Page = require('../Comps/Page').default

describe('Mock', () => {

    it('Mock', () => {
        const wrapper = mount(Page)
        // wrapper.vm.treeData = {}
        wrapper.vm.$forceUpdate()
    })
})
