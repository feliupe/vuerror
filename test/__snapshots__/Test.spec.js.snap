// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mock Mockkk 1`] = `undefined`;
